sample-operators/webpage/src/main/java/io/javaoperatorsdk/operator/sample/WebPageReconciler.java

```diff
@@ -52,18 +52,22 @@ public WebPageReconciler(KubernetesClient kubernetesClient) {
   public Map<String, EventSource> prepareEventSources(EventSourceContext<WebPage> context) {
     var configMapEventSource =
         new InformerEventSource<>(InformerConfiguration.from(ConfigMap.class)
+            .withNamespacesInheritedFromController(context)
             .withLabelSelector(LOW_LEVEL_LABEL_KEY)
             .build(), context);
     var deploymentEventSource =
         new InformerEventSource<>(InformerConfiguration.from(Deployment.class)
+            .withNamespacesInheritedFromController(context)
```

Comment:
Yes, I think that would be a reasonable default imo. The thing is that when I removed the that `from` before, was because it was simply not used. But will add such method.

Reply:
pushed.